### PR TITLE
stackrox: OCP: move 4.12 jobs to 4.13

### PR DESCRIPTION
**Pull request by gavin-stackrox**
This PR removes OCP 4.12 from Stackrox master, nightly and release template CI. It does *not* remove it from release branch CI.

Note for reviewers: The existing 4.12 jobs replaced the 4.13 ones as 4.12 had more coverage regarding fips, sesonsor-integration and core bpf.

/uncc
/hold

### CODE REVIEW

**Comment by gavin-stackrox** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-ocp-4-13-fips-qa-e2e-tests pull-ci-stackrox-stackrox-master-ocp-4-13-qa-e2e-tests

**Comment by gavin-stackrox** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-ocp-4-13-fips-qa-e2e-tests pull-ci-stackrox-stackrox-release-x.y-ocp-4-13-fips-qa-e2e-tests

**Comment by msugakov** [comment]
This is https://issues.redhat.com/browse/ROX-17375

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-13.yaml [hunk 1]
```diff
@@ -41,57 +52,82 @@ tests:
...
     env:
-      TEST_SUITE: ocp-qa-e2e-tests
+      TEST_SUITE: ocp-ui-e2e-tests
     test:
     - ref: stackrox-e2e
-    workflow: ipi-aws
+    workflow: ipi-gcp
   timeout: 5h0m0s
-- as: ui-e2e-tests
+- as: fips-qa-e2e-tests
```
Comment (msugakov) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-13.yaml:66]: [Paying back with the same coin :-P ] Does it make sense to group `*qa-e2e-tests` next to each other?

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-13.yaml [hunk 2]
```diff
@@ -20,16 +20,27 @@ resources:
...
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-gcp
+  timeout: 5h0m0s
 - as: operator-e2e-tests
   optional: true
   run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|COLLECTOR_VERSION|EXPECTED_GO_VERSION)
   steps:
-    cluster_profile: aws
+    cluster_profile: gcp
```
[the remark this answers is not in the document]
Comment (gavin-stackrox) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-13.yaml:38]: Yes

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-13.yaml (hunk 1 above)
Comment (gavin-stackrox) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-13.yaml:66]: I would say Yes :) But does that fall on this PR? I used the same order in the original 4.12 to replace the previous order in 4.13.

**Review by BradLugo — APPROVED** [review]
/lgtm

**Pull request merged**